deconstructsigs.wdl

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2021 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+version 1.0
+
+task DeconstructSigs {
+    input {
+        File signaturesMatrix
+        File signaturesReference
+        String outputPath = "./signatures.rds"
+
+        Int timeMinutes = 15
+        String memory = "4G"
+        String dockerImage = "quay.io/biocontainers/r-deconstructsigs:1.9.0--r41hdfd78af_1"
+    }
+
+    command {
+        R --no-echo << EOF
+            library(deconstructSigs)
+            tumor <- read.table("~{signaturesMatrix}", check.names=F)
```

Comment:
R will by default remove special characters (eg. `>` or `[`) from column names and replace them with a `.`. So a column name like `A[A>C]A` would become `A.A.C.A`. It is safer in this case to disable that, since these character convey important information.